### ISSUE TITLE
Behavior of PointerEvent.fromElement/toElement

### ISSUE THREAD

**Issue by RByers** [w3c/pointerevents#167]
fromElement/toElement [are being spec'd](https://github.com/w3c/pointerevents/issues/605) and it [looks like](https://github.com/w3c/web-platform-tests/pull/4408#issuecomment-271921770) there are some oddities in the implementation of these properties for pointer events in both Chrome and Edge.  Once the UI Events spec text has landed, does PointerEvents need to override it at all?  Even if not, we probably need a test.

@mustaqahmed says he'll look into it

/cc @cvrebert @dtapuska

**Comment by NavidZ** [comment]
We talked about this issue in the call and @mustaqahmed is going to look into the usecase and if we already support the usecase that fromElement/toElement are needed for in existing pointerevents then we just set them to default null in pointerevents.

**Comment by mustaqahmed** [comment]
Looks like we have better (standard and consistent) alternatives for these fields even in `MouseEvent`: [`.target`](https://developer.mozilla.org/en-US/docs/Web/API/Event/target) and [`.relatedTarget`](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/relatedTarget).

Given that browsers supported `fromElement`/`toElement` only to match an old IE behavior, and they are still [not consistent](https://github.com/w3c/pointerevents/issues/605) after so many years, we should better avoid these fields in PointerEvent IMO.

I will send a PR to spec them as undefined for PointerEvent unless there is an objection.

**Comment by RByers** [comment]
Sounds perfect to me!  PE is our one chance this decade to weed out the legacy crap in input ;-)

**Comment by mustaqahmed** [comment]
Oh, btw, Edge already returns `null` for these attributes in PointerEvents, as noted [here](https://github.com/w3c/web-platform-tests/pull/4529).

**Comment by mustaqahmed** [comment]
Reopening since we need to new/updated tests.

**Comment by mustaqahmed** [comment]
The test has landed.

**Comment by mustaqahmed** [comment]
Opening the issue, the tests we added need to be separated out somehow.

**Comment by cvrebert** [comment]
<details>
<summary>(Mooted by the next comment)</summary>

~~As you've declared #605 a duplicate of this issue, then speccing `MouseEvent.fromElement/toElement` (independent of the trivial behavior in the `PointerEvent` subclass) has now presumably become part of this issue.~~

</details>

**Comment by mustaqahmed** [comment]
> As you've declared [#605](https://github.com/w3c/pointerevents/issues/605) a duplicate of this issue, then speccing `MouseEvent.fromElement/toElement` (independent of the trivial behavior in the `PointerEvent` subclass) has now presumably become part of this issue.

You are right: getting those fields defined in `MouseEvent` is its own problem. Reopening #605.

**Comment by cvrebert** [comment]
Cool, then it would be nice to also retitle this issue to clarify its narrowed scope, now that the UI Events merger makes it ambiguous.

**Comment by mustaqahmed** [comment]
Fixed the test at https://github.com/web-platform-tests/wpt/commit/4d0836c26a190bd859732a4132b14267cff5ad17 . The spec makes it optional, so the assert needed to be conditional.

Re-closing the bug.
